quirks/password-rules.json

```diff
@@ -659,6 +659,9 @@
     "sfwater.org": {
         "password-rules": "minlength: 10; maxlength: 30; required: digit; allowed: lower, upper, [!@#$%*()_+^}{:;?.];"
     },
+    "store.nintendo.co.uk": {
+        "password-rules": "minlength: 8; maxlength: 20;"
+    }
```

Comment:
This needs a trailing comma. The file isn’t currently valid JSON.

Reply:
Good spot! Thanks!